frontend-new/src/chat/Chat.tsx

```diff
@@ -882,20 +884,20 @@ return {
           {newConversationDialog && (
             <ConfirmModalDialog
               isOpen={newConversationDialog}
-              title="Start New Conversation?"
+              title={t("start_new_conversation_title")}
               content={
                 <>
-                  Once you start a new conversation, all messages from the current conversation will be lost forever.
+                  {t("start_new_conversation_content")}
                   <br />
                   <br />
-                  Are you sure you want to start a new conversation?
+                  {t("are_you_sure_start_new_conversation_content")}
                 </>
               }
               onCancel={() => setNewConversationDialog(false)}
               onConfirm={handleConfirmNewConversation}
               onDismiss={() => setNewConversationDialog(false)}
-              cancelButtonText="Cancel"
-              confirmButtonText="Yes, I'm sure"
+              cancelButtonText={t("cancel_button")}
```

Comment:
[Option] name at least should be `cancel_button_label`.
We might have even many labels. ... (TBC)